modules/60crypt-ssh/helper/unlock-reap-success.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-SYSTEMD_CRYPTSETUP="$(ps -C systemd-cryptsetup -o pid=)"
+SYSTEMD_CRYPTSETUP="$(pgrep systemd-cryptsetup)"
```

Comment:
If we're already relying on `pkill` to clean up `cryptroot-ask`, we might as well use it here. The entire script could just be

```sh
#!/bin/sh
pkill systemd-cryptsetup || pkill cryptroot-ask
```

Reply:
Agreed. That seems like a good idea.

I updated with a new commit, and it works well using both busybox 1.36.1 and procps-ng 3.3.16-r2 (both the Gentoo stable versions).